python/catkin/builder.py

```diff
@@ -554,13 +655,14 @@ def build_cmake_package(
         make_install_cmd = [last_env] + make_install_cmd
     run_command(make_install_cmd, build_dir, quiet)
 
-    # If we are installing, and a env.sh exists, don't overwrite it
-    if install and os.path.exists(prefix_destdir(os.path.join(install_target, 'env.sh'), destdir)):
+    env_script = 'env' + '.bat' if sys.platform == 'win32' else '.sh'
```

Comment:
This change broke non-Windows platforms: see #1000.

Reply:
my apologies for the regression!